airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -766,9 +767,18 @@ def _clean(self, event: dict[str, Any]):
         # Skip await_pod_completion when the event is 'timeout' due to the pod can hang
         # on the ErrImagePull or ContainerCreating step and it will never complete
         if event["status"] != "timeout":
-            self.pod = self.pod_manager.await_pod_completion(
-                self.pod, istio_enabled, self.base_container_name
-            )
+            try:
+                self.pod = self.pod_manager.await_pod_completion(
+                    self.pod, istio_enabled, self.base_container_name
+                )
+            except ApiException as e:
+                if e.status == 404:
+                    self.pod = None
+                    self.log.info(
```

Comment:
Make this warn

Reply:
Can you change it to warn (as suggested by @amoghrajesh ?) @antoniocorralsierra ?

Reply:
Yes @potiuk it is already done

Reply:
ah... looked at old version :)